app/routes/faqs/calculation.tsx

```diff
@@ -0,0 +1,22 @@
+import type { LanguageTag } from "@sozialhelden/core";
+import { useLoaderData } from "react-router";
+import { i18nContext } from "~/context";
+import { loadMarkdownDocument } from "~/utils/content";
+import type { Route } from "./+types/calculation";
+
+export async function loader({ context }: Route.LoaderArgs) {
+  const { languageTag } = context.get(i18nContext);
+  return loadMarkdownDocument("faqs/calculation", languageTag as LanguageTag);
+}
+
+export default function FaqCalculation() {
```

Comment:
suggestion (blocking): IMO, this should be an `[articleName].tsx` route or something that makes this code more DRY to avoid drift – or is there a better way?